docs/source/howto/interact.rst

```diff
@@ -0,0 +1,112 @@
+.. _how-to:interact:
+
+**************************
+How to interact with AiiDA
+**************************
+
+There are a variety of manners to interact with AiiDA:
+
+ * :ref:`Through the command line interface <how-to:interact-cli>`
+ * :ref:`Through interactive shells <how-to:interact-shell>`
+ * :ref:`Through interactive notebooks <how-to:interact-notebook>`
+ * :ref:`Through scripts <how-to:interact-scripts>`
+ * :ref:`Through the REST API <how-to:interact-restapi>`
+
+
+.. _how-to:interact-cli:
+
+Command line interface
+======================
+
+AiiDA comes with a command line interface called ``verdi``.
+The :ref:`reference:command-line` section gives an overview of all available commands.
+For more detailed information, refer to the topic section :ref:`topics:cli`.
+
+
+.. _how-to:interact-shell:
+
+Interactive shells
+==================
+
+AiiDA provides a Python API that can be used from an interactive shell, such as `IPython <https://ipython.org/>`_.
+To use AiiDA from an IPython shell, one just has to load the profile as follows:
+
+.. code-block:: ipython
+
+    In [1]: from aiida import load_profile
+
+    In [2]: load_profile()
+    Out[2]: <aiida.manage.configuration.profile.Profile at 0x7fccfd6c50a0>
+
+One can pass a particular profile name to :meth:`~aiida.load_profile`, otherwise the default profile is loaded.
+Alternatively, the command line interface ``verdi`` provides a handy shortcut:
+
+.. code-block:: console
+
+    $ verdi shell
```

Comment:
Since you speak about profiles above, maybe this is the right place to remind people that the standard way of switching profile with verdi (not only for `shell`) is with `verdi -p profilename shell`

EDIT: I see this is explained below - as I suggest below, I would swap the order, so how to run a script is explained first, and this includes also `verdi -p XXX run`

Reply:
I would keep the shell first, but I can move the explanation for the `-p` option to here as well. Would that work?